src/lib/OpenEXRCore/internal_coding.h

```diff
@@ -25,10 +25,10 @@
 #endif
 
 #if defined(__has_include)
-#    if __has_include(<x86intrin.h>)
```

Comment:
This may be a slightly more elegant solution to the recent Imath build failure on ARM64 macoOS  https://github.com/AcademySoftwareFoundation/Imath/pull/184. @kdt3rd, can you confirm this include logic is proper?

Reply:
I think we should land the first fix for M1 clang compilation, and then revisit this logic on top of that. This logic will become real spaghetti unless we approach it step by step I think.

Reply:
@patlefort, let's deal with the include/intrin.h as a separate issue, can you remove this section from the PR? The M1/clang portion is addressed independently in #1112. Thanks.